requirements_all.txt

```diff
@@ -1488,7 +1488,7 @@ pyialarm==1.9.0
 pyicloud==0.10.2
 
 # homeassistant.components.insteon
-pyinsteon==1.0.9
+pyinsteon==1.0.11
```

Comment:
Bump to 1.0.12 needed in order to fix python 3.9 issues

Reply:
This PR has already been merged, any other update for that library needs to come in a new PR after it's tested by the person opening up the PR.